integration-tests/contracts/ccipreader_test.go

```diff
@@ -861,7 +862,7 @@ func testSetupRealContracts(
 	toMockBindings map[cciptypes.ChainSelector][]types.BoundContract,
 	env changeset.DeployedEnv,
 ) ccipreaderpkg.CCIPReader {
-	db := pgtest.NewSqlxDB(t)
+	db := pgcommon.NewSqlxDB(t, pgcommon.DbUrlOrInMemory(t))
```

Comment:
TODO:  add a `NewSqlxDB(t)`to pgcommon, and rename this one as a helper it calls so that we don't have to fall back to "in memory". This shouldn't ever matter since the db url should be configured automatically in integration tests, but if it did it would be better to just fail here rather than attempting to run in-memory which wouldn't work.